content/800-data-platform/100-accelerate/300-concepts.mdx

```diff
@@ -127,49 +127,41 @@ This means that Accelerate will create a maximum of 5 connections to the databas
 
 The connection pool size can be configured to a value other than the default via the _database connection string_.
 
+![Update database connection string in Accelerate](./images/accelerate-update-database-connection-string.png)
+
 To adjust the connection pool size, you may add the `connection_limit` parameter to the database connection string. The value of `connection_limit` is the maximum number of connections that can be created to the database.
 
 For example, here is how you can set a connection pool size of 10:
 
-```no-copy
+```env no-copy
 postgresql://user:password@localhost:5432/db?connection_limit=10
 ```
 
-After adjusting your database connection string, you must disable and re-enable Accelerate via the Cloud Project settings UI using the new connection string for the changes to take effect.
-
-![Disable Accelerate option in Cloud Project](./images/disable-accelerate.png)
-
-This will give you a new Accelerate connection string that you will need to update your application with.
-
-Update your application with the new Accelerate connection string. Then re-generate Prisma Client by running the following command:
+After updating your application with the new Accelerate connection string re-generate Prisma Client by running the following command:
```

Comment:
Do you actually get a new Accerelate connection string when updating the origin connection string?

Even if so, I don't think there is a reason to re-generate the Client - updating the connection string in your env var should be enough - unless you have it hardcoded. Most users should not have that - so maybe we can skip this.

Reply:
Okay that makes sense to me.